security.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Scala Security Policy
+layout: inner-page-no-masthead
+permalink: /security/
+includeTOC: false
+---
+
+## Receiving Security Announcements
+
+Security announcements related to Scala are published to the ["Security Announcements" channel](https://users.scala-lang.org/c/security) on our discourse forum.
+
+Messages to this channel can only be posted by administrators, so it is very low traffic.
+To set up email notifications for new security announcements, read [this post](https://users.scala-lang.org/t/about-the-security-announcements-category).
+
+## Reporting Vulnerabilities
+
+We strongly encourage reporting security issues in Scala to our private mailing list before disclosing them in public.
+
+The email address for security related communication is `security@scala-lang.org`.
+Messages are delivered to the Scala Security Team, which includes people from EPFL, the Scala Center, VirtusLab and Lightbend.
+
+We strive to acknowledge reports within 24 hours.
```

Comment:
Extend this, e.g. to 3 days?

Reply:
More importantly: business days ;)
May 2 business days?